src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,110 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private static final int DEFAULT_CAPACITY = 16;
+    private Node<K, V>[] table;
+    private int size;
+    private double threshold;
+
+    public class Node<K, V> {
```

Comment:
```suggestion
    private static class Node<K, V> {
```
private classes should be after private and public methods